src/Symfony/Component/Uuid/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020-2020 Fabien Potencier
```

Comment:
```suggestion
Copyright (c) 2020 Fabien Potencier
```

Reply:
By the way, if you are looking into something entirely over-engineered that covers this issue, take a look at [`ergebnis/license`](https://github.com/ergebnis/license).